package.json

```diff
@@ -101,9 +101,9 @@
     "@storybook/addons": "^5.2.4",
     "@storybook/react": "^5.2.4",
     "@storybook/theming": "^5.2.4",
-    "@testing-library/jest-dom": "4.2.4",
-    "@testing-library/react": "9.3.2",
-    "@testing-library/user-event": "8.1.0",
+    "@testing-library/jest-dom": "^5.5.0",
```

Comment:
@jendowns I remember you initially locked the versions of React Testing Library here - could you provide insight as to whether updating would cause any impact on the rest of the testing suite?

Reply:
@SimonFinney @diego-codes Digging into the git history here, I found this comment from January: https://github.com/carbon-design-system/ibm-security/pull/337#discussion_r373161223

It looks like I was having issues where CircleCI was using a slightly older version of `user-event` that was causing my tests to fail in the pipeline, but locally everything passed because I had the updated version. 🤷‍♀️

Reply:
Yeah, I don't remember why I did this... I will rever this because I didn't need it.